### ISSUE TITLE
animation for mdl-tab-panel's

### ISSUE THREAD

**Issue by maku** [mseemann/angular2-mdl#280]
It would be really cool to have animations for mdl-tab-panel's (e.g. like in https://material.angularjs.org/latest/demo/tabs)

**Comment by mseemann** [comment]
Right. But https://getmdl.io/components/index.html#layout-section/tabs has no animations 🤔

**Comment by maku** [comment]
Yes I know but the transitions from one tab to another are so "hard". Especially when you have more content on the tab. The example on https://material.angularjs.org/latest/demo/tabs looks so much nicer...
